R/model_id_merge.R

```diff
@@ -0,0 +1,94 @@
+#' Merge/Split model output tbl `model_id` column
+#'
+#' @inheritParams as_model_out_tbl
+#' @param sep character string. Character used as separator when concatenating
+#' `team_abbr` and `model_abbr` values into a single `model_id` string or splitting
+#' `model_id` into component `team_abbr` and `model_abbr`. When splitting, if
+#' multiple instances of the separator exist in a `model_id` stringing,
```

Comment:
another option here would be to error out rather than picking one occurrence to split on. We have been careful in the hubs to validate that the separation character is not allowed in either `team_abbr` or `model_abbr`.

Reply:
Happy to implement whatever you prefer. Personally I feel splitting on the first one a sensible default (as it reflects basic regex pattern matching), especially combined with strong validation on the hub side. However, should the data have been corrupted in any way between the hub and `as_model_out_tbl` an error would indeed at least alert to that.

Let me know what you and @elray1 prefer.

Reply:
I think my preference would be to have it error if splitting is ambiguous? not a strongly held preference.

Reply:
I don't have a strong opinion here.

Reply:
Implemented in 3c16503d3f862021fda9241dc5ed094711026d6f